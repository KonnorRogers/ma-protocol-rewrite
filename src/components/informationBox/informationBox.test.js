import React from "react"
import { render } from "@testing-library/react"
import InformationBox from "./informationBox.js"

test("Renders without error", () => {
  const { asFragment } = render(<InformationBox borderColor="red" />)
  expect(asFragment).toMatchSnapshot()
})
