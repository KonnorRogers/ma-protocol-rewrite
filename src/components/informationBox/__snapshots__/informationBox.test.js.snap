// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders without error 1`] = `
<DocumentFragment>
  .emotion-0 {
  margin: 0.75rem;
  padding: 1.5rem 2rem;
  border-radius: 12px;
  border: 2px solid red;
  text-align: center;
  font-size: 1rem;
}

@media only screen and (min-width:640px) {
  .emotion-0 {
    font-size: 1.1rem;
  }
}

.emotion-0 {
  margin: 0.75rem;
  padding: 1.5rem 2rem;
  border-radius: 12px;
  border: 2px solid red;
  text-align: center;
  font-size: 1rem;
}

@media only screen and (min-width:640px) {
  .emotion-0 {
    font-size: 1.1rem;
  }
}

<div
    class="emotion-0"
  />
</DocumentFragment>
`;
